components/FinancialSummary/SuccessStories.js

```diff
@@ -0,0 +1,66 @@
+import Heading from "../typography/Heading"
+import Paragraph from '../typography/Paragraph'
```

Comment:
Either use a single quote or double quotes to denote the path for importing components. We usually use single quotes in majority of files so kindly avoid using the double quotes here. Applicable for all files.